query/search.js

```diff
@@ -102,7 +102,10 @@ function generate( params ){
   // add search condition to distance query
   query.query.filtered.query.bool.must.push({ 
     'match': {
-      'name.default': input
+      'name.default': {
+        'query': input,
+        'analyzer': 'peliasOneEdgeGram'
```

Comment:
what do you think about API being more tightly coupled with schema? I don't like the fact that we just have strings like this around without any validation that corresponding schema version contains them. I'm working on coupling schema with API for admin fields.